.circleci/config.yml

```diff
@@ -24,29 +24,33 @@ jobs:
       - run:
           name: Copy RediSearch
           command: |
-            docker run --rm --entrypoint cat redislabs/redisearch:edge  /usr/lib/redis/modules/redisearch.so > redisearch.so
+            docker run --rm --entrypoint cat redislabs/redismod:edge /usr/lib/redis/modules/redisearch.so > redisearch.so
             chmod 755 redisearch.so
+            docker run --rm --entrypoint cat redislabs/redismod:edge /usr/lib/redis/modules/rejson.so > rejson.so
+            chmod 755 rejson.so
+
       - run:
           name: Run RediSearch with tls support
           command: |
             docker run -d -v $(pwd)/redisearch.so:/data/redisearch.so \
+                       -v $(pwd)/rejson.so:/data/rejson.so \
                        -v $(pwd)/redis/tests/tls/:/data \
                        -p 6379:6379 redis redis-server --tls-port 6379 --port 0  \
                        --tls-cert-file /data/redis.crt  \
                        --tls-key-file /data/redis.key  \
                        --tls-ca-cert-file /data/ca.crt \
-                       --tls-auth-clients no  --loadmodule /data/redisearch.so
+                       --tls-auth-clients no --loadmodule /data/redisearch.so --loadmodule /data/rejson.so
       - run:
           name: Run Examples
           command: |
             make examples TLS_CERT=redis/tests/tls/redis.crt \
                           TLS_KEY=redis/tests/tls/redis.key \
                           TLS_CACERT=redis/tests/tls/ca.crt
 
-  build: # test with redisearch:edge
+  build-latest:
```

Comment:
```suggestion
  build: # test with redismod:latest
```

Reply:
@AvitalFineRedis we need to keep the job named `build` given it's a required build step for the PRs